packages/docs/docs/lottie/getlottiemetadata.md

```diff
@@ -0,0 +1,211 @@
+          k: [
+            {
+              i: { x: 0.833, y: 0.833 },
+              o: { x: 0.167, y: 0.167 },
+              t: 0,
+              s: [979.401, 1368, 0],
+              to: [0, -138, 0],
+              ti: [0, 138, 0],
+            },
+            { t: 58.0000023623884, s: [979.401, 540, 0] },
+          ],
+          ix: 2,
+          l: 2,
+        },
+        a: { a: 0, k: [517.365, 112.096, 0], ix: 1, l: 2 },
+        s: { a: 0, k: [100, 100, 100], ix: 6, l: 2 },
+      },
+      ao: 0,
+      shapes: [
+        {
+          ty: "gr",
+          it: [
+            {
+              ty: "rc",
+              d: 1,
+              s: { a: 0, k: [425.883, 425.883], ix: 2 },
+              p: { a: 0, k: [0, 0], ix: 3 },
+              r: { a: 0, k: 98, ix: 4 },
+              nm: "Rectangle Path 1",
+              mn: "ADBE Vector Shape - Rect",
+              hd: false,
+            },
+            {
+              ty: "st",
+              c: { a: 0, k: [1, 1, 1, 1], ix: 3 },
+              o: { a: 0, k: 100, ix: 4 },
+              w: { a: 0, k: 2, ix: 5 },
+              lc: 1,
+              lj: 1,
+              ml: 4,
+              bm: 0,
+              nm: "Stroke 1",
+              mn: "ADBE Vector Graphic - Stroke",
+              hd: false,
+            },
+            {
+              ty: "fl",
+              c: { a: 0, k: [0, 0.468933612108, 1, 1], ix: 4 },
+              o: { a: 0, k: 100, ix: 5 },
+              r: 1,
+              bm: 0,
+              nm: "Fill 1",
+              mn: "ADBE Vector Graphic - Fill",
+              hd: false,
+            },
+            {
+              ty: "tr",
+              p: { a: 0, k: [494.618, 123.481], ix: 2 },
+              a: { a: 0, k: [0, 0], ix: 1 },
+              s: { a: 0, k: [100, 100], ix: 3 },
+              r: { a: 0, k: 0, ix: 6 },
+              o: { a: 0, k: 100, ix: 7 },
+              sk: { a: 0, k: 0, ix: 4 },
+              sa: { a: 0, k: 0, ix: 5 },
+              nm: "Transform",
+            },
+          ],
+          nm: "Rectangle 2",
+          np: 3,
+          cix: 2,
+          bm: 0,
+          ix: 1,
+          mn: "ADBE Vector Group",
+          hd: false,
+        },
+      ],
+      ip: 0,
+      op: 90.0000036657751,
+      st: 0,
+      ct: 1,
+      bm: 0,
+    },
+    {
+      ddd: 0,
+      ind: 2,
+      ty: 1,
+      nm: "White Solid 1",
+      sr: 1,
+      ks: {
+        o: { a: 0, k: 100, ix: 11 },
+        r: { a: 0, k: 0, ix: 10 },
+        p: { a: 0, k: [960, 540, 0], ix: 2, l: 2 },
+        a: { a: 0, k: [960, 540, 0], ix: 1, l: 2 },
+        s: { a: 0, k: [100, 100, 100], ix: 6, l: 2 },
+      },
+      ao: 0,
+      sw: 1920,
+      sh: 1080,
+      sc: "#ffffff",
+      ip: 0,
+      op: 90.0000036657751,
+      st: 0,
+      bm: 0,
+    },
+  ],
+  markers: [],
+};
+
+// @filename: Animation.tsx
+import { animationData } from "./animation";
+
+// ---cut---
+import { getLottieMetadata } from "@remotion/lottie";
```

Comment:
Although `@remotion/lottie` is declared on the `package.json`, this is still throwing an error for me locally. It happens on every package when the package is imported. Does it happen for you? Any idea why?

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/13774309/183088461-208de6f4-aea9-43ac-b69e-0723ca5ca82b.png)


</details>

Reply:
Hmm, it needs a `pnpm i` from the root again even though this is a local dependency and depending on if caching went wrong it might also need a `pnpm run clean` from the root